angelsindustries/prototypes/overrides/overhaul-nuclear-power.lua

```diff
@@ -47,9 +47,10 @@ if angelsmods.industries.overhaul then
         localised_name = {"recipe-name.uranium-fuel-cell"},
         category = "centrifuging",
         ingredients = {
+          {"!!"},
           {type = "item", name = "uranium-235", amount = 1},
-          {type = "item", name = "uranium-238", amount = 29}
-          --[[{type="item",name="angels-plate-lead",amount=30}]]
+          {type = "item", name = "uranium-238", amount = 29},
+          {type = "item", name = "angels-plate-iron",amount = 15}

```

Comment:
Shouldn't this be iron-plate?

Reply:
resolved in https://github.com/Arch666Angel/mods/pull/272/commits/44614af7c5a752ceef52f1850ce2f26d55e2554f